presidio-anonymizer/presidio_anonymizer/anonymizers/hash.py

```diff
@@ -1,19 +1,37 @@
 """Hashes the PII text entity."""
+from hashlib import sha256, sha512, md5
 from presidio_anonymizer.anonymizers import Anonymizer
+from presidio_anonymizer.anonymizers.validators import validate_parameter_in_range
 
 
 class Hash(Anonymizer):
-    # TODO implement + test
-    """Hash given text with sha256 algorithm."""
+    """Hash given text with sha256/sha512/md5 algorithm."""
+
+    HASH_TYPE = "hash_type"
+    SHA256 = "sha256"
+    SHA512 = "sha512"
+    MD5 = "md5"
 
     def anonymize(self, text: str = None, params: dict = None) -> str:
         """
         Hash given value using sha256.
 
         :return: hashed original text
         """
-        return params.get("old_text")
+        hash_type = params.get(self.HASH_TYPE, self.SHA256)
```

Comment:
are we documenting that the default is sha265?

Reply:
done